### PR TITLE
Added "to connect", "quotation" and "RFQ".

### PR DESCRIPTION
**Pull request by armandas**

### CODE REVIEW

File: ReadMe.md [hunk 1]
```diff
@@ -232,6 +233,8 @@ Office | 事務所 | じむしょ |
 In-house | 社内 | しゃない |
 External, 3rd party | 外部 | がいぶ |
 Suspend, discontinue | 中止 | ちゅうし |
+Quotation | 見積 | みつもり |
```
Comment (ronkorving) [inline ReadMe.md]: I just learned that these also appear:

> 見積もり
> 見積り (my wife thinks this one is the more common of the two)

Could you perhaps add these to the notes field? Eg: "Also: 見積り or 見積もり)"
Thanks!

**Comment by ronkorving** [comment]
Thanks for the contribution! I added one comment, hope you don't mind addressing that. Thanks!

**Comment by armandas** [comment]
I have added the alternative forms, in the order they appear on jisho.org.

**Comment by ronkorving** [comment]
Thanks!

**Comment by ronkorving** [comment]
Wow, that's a big addition :)
I will merge it as-is, because reviewing it will take too long. If there are any issues with it, someone can make a pull request for it :)
Thanks!

**Comment by armandas** [comment]
Oops, sorry! I was meaning to create a separate PR for the EE topic, as I wasn't sure if fitted in this dictionary.

Anyway, thanks for adding my changes!

**Comment by ronkorving** [comment]
That would've made sense, but no worries :) I think it's a cool addition :)

**Pull request merged**